.travis.yml

```diff
@@ -1,14 +1,29 @@
 ---
-
-language: python
+language: "python"
+dist: "xenial"
+os: "linux"
 
 python:
   - "3.6"
-
-sudo: false
+  - "3.7"
+  - "3.8"
 
 install:
-  - pip install tox
+  - "pip install virtualenv"
```

Comment:
Could we just use the builtin `venv` module here?

`python -m venv .venv`?

Reply:
I tried that at some point, and it did not like it. Travis should have the default pip with a version that supports installing cryptography soon, and this can go away.